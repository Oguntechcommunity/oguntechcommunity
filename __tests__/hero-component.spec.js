import * as React from "react";
import { shallow } from "enzyme";
import Hero from "../pages/components/hero/hero-component"

describe("Pages", () => {
	describe("Hero Component", () => {

        const wrapper = shallow(<Hero />);

		it("should match snapshot", () => {

			expect(wrapper).toMatchSnapshot();
        });

        it("Should have an empty name, portfolio, email, job", () => {
            //Optionally test to check if name, portfolio, email, job are empty strings on 

            expect(wrapper.state({name: " "}));
            expect(wrapper.state({portfolio: " "}));
            expect(wrapper.state({email: " "}));
            expect(wrapper.state({job: " "}));

            wrapper.setState({name: "Niyi"});
            wrapper.setState({portfolio: "https://www.google.com/"});
            wrapper.setState({email: "john@gmail.com"});
            wrapper.setState({job: "developer"});
        });

        it("Should have an value (setState)", () => {
            //Optionally test to check if name, portfolio, email, job are empty strings on 

        });
	});
});
