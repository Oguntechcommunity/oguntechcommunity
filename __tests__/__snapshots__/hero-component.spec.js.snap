// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages Hero Component should match snapshot 1`] = `ShallowWrapper {}`;
